core/services/relay/evm/types/types.go

```diff
@@ -20,11 +20,47 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/utils"
 )
 
+type RelayOpts struct {
+	// TODO BCF-2508 -- should anyone ever get the raw config bytes that are embedded in args? if not,
+	// make this private and wrap the arg fields with funcs on RelayOpts
+	relaytypes.RelayArgs
+	c *RelayConfig
+}
+
+type ChainReaderConfig struct {
+	// key being contract name
+	ChainContractReaders map[string]ChainContractReader `json:"chainContractReaders"`
+
+	// ..reserved for any global config options chainreader might need..
```

Comment:
Does this comment add value? If so, is there somewhere else that makes sense to put it? because the tooling drops it on the floor since it is not associated with anything.

Reply:
Yeah no need for this comment, should be pretty obvious for anyone that wants to make changes